libs/langchain/langchain/vectorstores/pinecone.py

```diff
@@ -16,6 +17,15 @@
 logger = logging.getLogger(__name__)
 
 
+def chunks(iterable: Iterable, batch_size: int = 100) -> Iterable:
+    """A helper function to break an iterable into chunks of size batch_size."""
```

Comment:
NIT: should we add recommendations around when / where to use this in the docstring you added?

Reply:
Yeah, I'll add docs. It's missing on some other places too.
First I need to see if there is already similar function somewhere in the lagnchain since it looks like simple util function.